File: MyHeart.Web/ClientApp/models/professionInfo/bonds/MedicamentGroupDiseaseContraindicationDto.ts
import DiseaseDto from "../DiseaseDto";

export default class MedicamentGroupDiseaseContraindicationDto {
    medicamentGroupId: number;
    diseaseId: number;
    disease: DiseaseDto;

    constructor(medicamentGroupId: number, disease: DiseaseDto) {
        this.medicamentGroupId = medicamentGroupId;
        this.diseaseId = disease.id;
        this.disease = disease;
    }
}

File: MyHeart.DTO/ProfessionInformation/MedicamentGroupBonds/MedicamentGroup_Disease_ContraindicationDTO.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace MyHeart.DTO.ProfessionalInformation
{
    public class MedicamentGroup_Disease_ContraindicationDTO : BaseEntityDTO
    {
        public int MedicamentGroupId { get; set; }
        public int DiseaseId { get; set; }

    }
}
